src/main/kotlin/day5/Wizard.kt

```
@@ -0,0 +1,54 @@
+
+    val deadHp = 0
+
+    init {
+        this.name = name
+        this.wand = wand
+        this.hp = hp
+    }
+
+    fun attack() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)